include/mgos_pcf857x.h

```diff
@@ -18,13 +18,16 @@
 #include "mgos.h"
 #include "mgos_gpio.h"
 #include "mgos_i2c.h"
+#include "mgos_timers.h"
 
 #ifdef __cplusplus
 extern "C" {
 #endif
 
 struct mgos_pcf857x;
 
+struct mgos_pcf857x_gpio_blink_state;
```

Comment:
this is not meant to be a public struct (the state is never retrieved/used/serviceable by users of this library). Therefore, move to src/_internal.h